contracts/QuestFactory.sol

```diff
@@ -85,6 +86,7 @@ contract QuestFactory is Initializable, LegacyStorage, OwnableRoles, IQuestFacto
         referralFee = referralFee_;
         mintFee = mintFee_;
         referralRewardTimestamp = block.timestamp;
+        referralRewardFee = 250; // in BIPS
```

Comment:
The `referralFee` on line 86 refers to the referral fees on the claim fee (which is currently set to 0 ETH)